src/content/docs/en/pages/guides/edge-application/ea-enforce-hls.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: How to enforce HLS cache for live streaming delivery
+description: This guide covers the step-by-step to enforce HLS cache, managing chunks and playlists caching and setting up rules engines rules.
+meta_tags: architecture, content delivery, hls, edge computing
+namespace: docs_guides_enforce_hls_cache
+permalink: /documentation/products/guides/enforce-hls-cache/
+---
+
+import Button from '~/components/Button.astro';
+import Tabs from '~/components/tabs/Tabs'
+
+:::caution[important]
+Azion has two user interfaces: Real-Time Manager and Console, which is in Preview stage. Currently, Console is only available for Developer plans and new accounts. Follow the steps according to the user interface you're using.
+:::
+
+Azion Edge Platform allows you to deliver live streaming content in **HLS format**, being able to easily self-provision and configure your cache policies. This guide covers the step-by-step to enforce HLS cache, managing chunks and playlists caching and setting up rules engines rules.
```

Comment:
*Maybe* linking keywords in the first paragraph would be better than creating a button to use case page.